rasa/core/actions/action.py

```diff
@@ -77,13 +79,22 @@
 import rasa.shared.utils.io
 from rasa.utils.endpoints import EndpointConfig, ClientResponseError
 
+
 if TYPE_CHECKING:
     from rasa.core.nlg import NaturalLanguageGenerator
     from rasa.core.channels.channel import OutputChannel
     from rasa.shared.core.events import IntentPrediction
 
 logger = logging.getLogger(__name__)
 
+try:
+    OMIT_DOMAIN = int(os.environ.get(ENV_ACTION_OMIT_DOMAIN, "0"))
```

Comment:
should move to endpoint config

Reply:
I have to admit that I did not dare to touch that, because it is distributed via Rasa X for enterprise customers which is kind of closed source.
Any ideas how to integrate that? Is the endpoints JSON object passed "as is"?

Reply:
According to 
https://rasa.com/docs/rasa-enterprise/installation-and-setup/configure/add-action-server/#starting-an-action-server-using-the-rasa-enterprise-helm-chart
the configuration is kind of automatic in _enterprise_ so I have no idea how to tell the helm chart to add keys/values to the configuration of the endpoint.

Reply:
I was pointed to this
https://github.com/RasaHQ/rasa-x-helm/blob/rasa-x-4.5.12/charts/rasa-x/templates/rasa-config-files-configmap.yaml#L55-L57
by my colleagues. So it needs to have a placeholder in the template?

Reply:
@tmbo - could this be part of the `action_server` config in endpointConfig where we can add `omit_domain`